packages/react-tabs/src/components/Tab/useTab.ts

```diff
@@ -45,23 +54,28 @@ export const useTab_unstable = (props: TabProps, ref: React.Ref<HTMLElement>): T
     };
   }, [onRegister, onUnregister, innerRef, value]);
 
+  const iconShorthand = resolveShorthand(icon);
+  const contentShorthand = resolveShorthand(content, { required: true, defaultProps: { children: props.children } });
```

Comment:
```suggestion
```

nit: you don't reuse these, so was there anything wrong with the previous syntax ?

Reply:
I tried a different approach that didn't work out and this is leftovers from trying that (facepalm).  Good catch!